doc/source/skips/0-skip-process.rst

```diff
@@ -0,0 +1,301 @@
+original obsolete. The ``Replaced-By`` and ``Replaces`` headers
+should be added to the original and new SKIPs respectively.
+
+Process SKIPs may also have a status of ``Active`` if they are never
+meant to be completed, e.g. SKIP 0 (this SKIP).
+
+
+How a SKIP becomes Accepted
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A SKIP is ``Accepted`` by consensus of all interested contributors. We
+need a concrete way to tell whether consensus has been reached. When
+you think a SKIP is ready to accept, send an email to the
+scikit-image mailing list with a subject like:
+
+  Proposal to accept SKIP #<number>: <title>
+
+In the body of your email, you should:
+
+* link to the latest version of the SKIP,
+
+* briefly describe any major points of contention and how they were
+  resolved,
+
+* include a sentence like: "If there are no substantive objections
+  within 7 days from this email, then the SKIP will be accepted; see
+  SKIP 0 for more details."
+
+For an equivalent example in the NumPy library, see: https://mail.python.org/pipermail/numpy-discussion/2018-June/078345.html
+
+After you send the email, you should make sure to link to the email
+thread from the ``Discussion`` section of the SKIP, so that people can
+find it later.
+
+Generally the SKIP author will be the one to send this email, but
+anyone can do it – the important thing is to make sure that everyone
+knows when a SKIP is on the verge of acceptance, and give them a final
+chance to respond. If there's some special reason to extend this final
+comment period beyond 7 days, then that's fine, just say so in the
+email. You shouldn't do less than 7 days, because sometimes people are
+travelling or similar and need some time to respond.
+
+In general, the goal is to make sure that the community has consensus,
+not provide a rigid policy for people to try to game. When in doubt,
+err on the side of asking for more feedback and looking for
+opportunities to compromise.
+
+If the final comment period passes without any substantive objections,
+then the SKIP can officially be marked ``Accepted``. You should send a
+followup email notifying the list (celebratory emoji optional but
+encouraged 🎉✨), and then update the SKIP by setting its ``:Status:``
+to ``Accepted``, and its ``:Resolution:`` header to a link to your
+followup email.
+
+If there *are* substantive objections, then the SKIP remains in
+``Draft`` state, discussion continues as normal, and it can be
+proposed for acceptance again later once the objections are resolved.
+
+In unusual cases, the `scikit-image Steering Council`_ may be asked to decide
+whether a controversial SKIP is ``Accepted``.
+
+
+Maintenance
+^^^^^^^^^^^
+
+In general, Standards track SKIPs are no longer modified after they have
+reached the Final state as the code and project documentation are considered
+the ultimate reference for the implemented feature.
+However, finalized Standards track SKIPs may be updated as needed.
```

Comment:
This sentence seems to contradict the previous paragraph.

Reply:
@stefanv this is [directly lifted from NEP-0](https://numpy.org/neps/nep-0000.html#maintenance). Would you like to clarify with @jarrodmillman what this text was *supposed* to say? =)

Reply:
I'll check with Jarrod.

Reply:
Basic gist: "These are typically not modified after reaching Final, but may be under special circumstances."